ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -113,6 +118,12 @@ public Claim from(CCDCase ccdCase) {
             settlement = settlementMapper.from(ccdCase.getSettlement());
         }
 
+        List<String> features = null;
+        if (ccdCase.getFeatures() != null) {
+            features = Arrays.asList(ccdCase.getFeatures().split(","));
+        }
+
+
         return new Claim(
```

Comment:
I know its not part of your change. Should we use the builder rather than the constructor with nulls. If you have time can be fixed.

Reply:
https://github.com/hmcts/cmc-claim-store/pull/444#discussion-diff-210218972R137